### DISCUSSION TITLE
記事のタイトルに#を含めたい

### DISCUSSION THREAD

**Discussion by clcl777** [growilabs/growi#7829] [Ideas / Requests] [answered]
C#に関する記事を書こうと思ったら、「Invalid characters on path」とエラーが出ます。タイトルにも#をつけられるようにして欲しいです。

**Comment by yuki-takei** [comment, marked as answer]
GROWI ではページの URL を入力してアクセスする際、一般的にはページの ID で一意に決められた permalink を用いますが、同時にページのパスをエイリアス(別名)の URL として取り扱うことができます。

**例:**
permalink => https://example.com/58a4569921a8424d00a1aa0e
alias link => https://example.com/Sandbox/Math

この仕様のため、URL を構成する特殊文字として利用されるキャラクター、`/` , `?` , `&`, `#` などをページパスに利用することはできません。

## 代替案
全角の `＃` を使うなどしてください。

## Note

特殊文字をタイトルに含めるのを許容しつつ利用時はエスケープする案もあるとは思うが、対応するためのコードが複雑になることが予想される。ユーザーコミュニティによって工数・影響範囲検討が本格的に行われ、修正まで漕ぎ着けらるようであればそれは止めはしない。
